lib/plugins/exchange/ccxt.js

```diff
@@ -0,0 +1,51 @@
+var ccxt = require('ccxt')
+const coinUtils = require('../../coin-utils')
+const common = require('../common/ccxt')
+const kraken = require('./kraken')
+const bitstamp = require('./bitstamp')
+const itbit = require('./itbit')
+const consts = require('./consts')
+
+const ALL_EXCHANGES = {
+  kraken,
+  bitstamp,
+  itbit
+}
+
+function trade (side, account, cryptoAtoms, fiatCode, cryptoCode, exchangeName) {
+  try {
+    const exchangeConfig = ALL_EXCHANGES[exchangeName]
+
+    if (!exchangeConfig) throw Error('no exchange')
+    if (exchangeConfig.isConfigValid && !exchangeConfig.isConfigValid(account)) throw Error('Invalid config')
+
+    const exchange = new ccxt[exchangeName](exchangeConfig.loadConfig(account))

```

Comment:
we can probably make loadConfig default to noop, it's a identity function so it will just return `account`

```js
const { isConfigValid, loadConfig = _.noop, loadOption, AMOUNT_PRECISION, ORDER_TYPE} = exchangeConfig
```

Reply:
Also it looks like you're only using `exchange` on line 29 and 31, might as well move this line closer to that and add a empty line before it.